app/src/main/java/org/glucosio/android/activity/RemindersActivity.java

```diff
@@ -46,18 +53,38 @@ protected void onCreate(Bundle savedInstanceState) {
 
         addFab.setOnClickListener(new View.OnClickListener() {
             @Override
-            public void onClick(View view) {
-                // Open Time Picker on FAB click
-                boolean is24HourFormat = android.text.format.DateFormat.is24HourFormat(getApplicationContext());
-                Calendar cal = presenter.getCalendar();
-
-                TimePickerDialog tpd = TimePickerDialog.newInstance(
-                        RemindersActivity.this,
-                        cal.get(Calendar.HOUR_OF_DAY),
-                        cal.get(Calendar.MINUTE),
-                        is24HourFormat);
-                tpd.show(getFragmentManager(), "Timepickerdialog");
-            }
+            public void onClick(final View view) {
+
+                AlertDialog.Builder builder = new AlertDialog.Builder(ReminderActivity);
```

Comment:
IMHO it looks bad to have just 2 EditText in one dialog. And you still need to open the time picker dialog anyway.